docs/superpowers/plans/2026-05-29-cli-supabase-perf-tracking.md

```
@@ -0,0 +1,939 @@
+Run: `bun test/test-supabase-perf.mjs`
+Expected: PASS — `✅ supabase-perf tests passed`.
+
+- [ ] **Step 5: Register the test in package.json**
+
+In `cli/package.json`, add a script next to the other `test:analytics*` scripts:
+
+```json
+"test:supabase-perf": "bun test/test-supabase-perf.mjs",
+```
+
+Then add ` && bun run test:supabase-perf` to the long `"test"` script, immediately after `bun run test:analytics-org-resolver`.
```

markdownlint-cli2 (0.22.1)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)